<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderResultWidget</class>
 <widget class="QWidget" name="RenderResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="selectModeButton">
         <property name="text">
          <string>Select an instance of a model to add:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="modelComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="addInstanceButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="GUI::OpenGL::GLWidget" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="undoButton">
            <property name="text">
             <string>Undo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="selectedModelLabel">
          <property name="text">
           <string>Selected Model:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="translationLabel">
            <property name="text">
             <string>Translation:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="translationXRadioButton">
            <property name="text">
             <string>X-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="translationXLineEdit"/>
          </item>
          <item>
           <widget class="QRadioButton" name="translationYRadioButton">
            <property name="text">
             <string>Y-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="translationYLineEdit"/>
          </item>
          <item>
           <widget class="QRadioButton" name="translationZRadioButton">
            <property name="text">
             <string>Z-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="translationZLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="rotationLabel">
            <property name="text">
             <string>Rotation:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="rotationXRadioButton">
            <property name="text">
             <string>X-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rotationXLineEdit"/>
          </item>
          <item>
           <widget class="QRadioButton" name="rotationYRadioButton">
            <property name="text">
             <string>Y-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rotationYLineEdit"/>
          </item>
          <item>
           <widget class="QRadioButton" name="rotationZRadioButton">
            <property name="text">
             <string>Z-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rotationZLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Scale</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="scaleXRadioButton">
            <property name="text">
             <string>X-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="scaleXLineEdit"/>
          </item>
          <item>
           <widget class="QRadioButton" name="scaleYRadioButton">
            <property name="text">
             <string>Y-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="scaleYLineEdit"/>
          </item>
          <item>
           <widget class="QRadioButton" name="scaleZRadioButton">
            <property name="text">
             <string>Z-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="scaleZLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUI::OpenGL::GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>GUI/OpenGL/GLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
